wagtail/images/blocks.py

```diff
@@ -51,3 +56,133 @@ def htmldiff(self):
                 "image_b": self.val_b,
             },
         )
+
+
+class ImageBlock(StructBlock):
+    image = ImageChooserBlock(required=True)
+    decorative = BooleanBlock(
+        default=False, required=False, label="Image is decorative"
```

Comment:
```suggestion
        default=False, required=False, label=_("Image is decorative")
```

Reply:
I take it I will have to import through `django.utils.translation import gettext as _ `, right?